_overviews/core/collections-migration-213.md

```diff
@@ -0,0 +1,330 @@
+---
+layout: singlepage-overview
+title: Migrating a Project to Scala 2.13's Collections
+permalink: /overviews/core/:title.html
+---
+
+This document describes the main changes for collection users that migrate to Scala 2.13 and shows
+how to cross-build projects with Scala 2.11 / 2.12 and 2.13.
+
+For an in-depth overview of the Scala 2.13 collections library, see the [collections guide]({{ site.baseurl }}/overviews/collections-2.13/introduction.html). The implementation details of the 2.13 collections are explained in the document [the architecture of Scala collections]({{ site.baseurl }}/overviews/core/architecture-of-scala-213-collections.html).
+
+## What are the major changes?
+
+See the list at the top of the [2.13.0-M4 release notes](https://github.com/scala/scala/releases/tag/v2.13.0-M4).
+
+## What are the breaking changes?
+
+The following table summarizes the breaking changes:
+
+| Description | Old Code | New Code | Automatic Migration Rule |
+| ----------- | -------- | -------- | ------------------------ |
+| Method `to[C[_]]` has been removed (it might be reintroduced but deprecated, though) | `xs.to[List]` | `xs.to(List)` | `NewCollections`, `CrossCompat` |
+| `mutable.Map` no longer have an `updated` method | `mutable.Map(1 -> 2).updated(1, 3)` | `mutable.Map(1 -> 2).clone() += 1 -> 3` | `NewCollections`, `CrossCompat` |
+| `mapValues` and `filterKeys` now return a `MapView` instead of a `Map` | `kvs.mapValues(f)` | `kvs.mapValues(f).toMap` | `RoughlyMapValues` |
+| `Iterable` no longer has a `sameElements` operation | `xs1.sameElements(xs2)` | `xs1.iterator.sameElements(xs2)` | `NewCollections`, `CrossCompat` |
+| `collection.breakOut` no longer exists | `val xs: List[Int] = ys.map(f)(collection.breakOut)` | `val xs = ys.iterator.map(f).to(List)` | `NewCollections` |
+| `zip` on `Map[K, V]` now returns an `Iterable` | `map.zip(iterable)` | `map.zip(iterable).toMap` | `Experimental` |
+| `ArrayBuilder.make` does not accept parens anymore | `ArrayBuilder.make[Int]()` | `ArrayBuilder.make[Int]` | `NewCollections`, `CrossCompat` |
```

Comment:
The names of the migration rules have changed. We now have `Collection213Upgrade` and `Collections213CrossCompat`, instead of `NewCollections` and `CrossCompat`, respectively.

Reply:
done